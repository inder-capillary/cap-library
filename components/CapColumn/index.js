/**
* CapColumn
*/
import React from "react";
import PropTypes from 'prop-types';
import { Col } from "antd";
import "./_capColumn.scss";
const classNames = require('classnames');

export default class CapColumn extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    const {className, children, ...rest} = this.props;
    return (
      <Col {...rest} className={classNames("cap-column-v2", className)}>
        {React.Children.toArray(children)}
      </Col>
    );
  }
}

CapColumn.propTypes = {
  children: PropTypes.any,
  className: PropTypes.string,
};
