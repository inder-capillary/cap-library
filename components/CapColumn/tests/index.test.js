import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars
// import {mount} from 'enzyme';

import CapColumn from '../index';

describe('<CapColumn />', () => {
  it('CapColumn renders correctly', () => {
    const tree = renderer.create(<CapColumn />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
