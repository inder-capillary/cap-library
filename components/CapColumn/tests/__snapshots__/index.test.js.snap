// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapColumn /> CapColumn renders correctly 1`] = `
<div
  className="cap-column-v2"
/>
`;
